kasa/json.py

```diff
@@ -11,5 +11,8 @@ def dumps(obj, *, default=None):
 except ImportError:
     import json
 
-    dumps = json.dumps
+    def dumps(obj, *, default=None):
+        """Dump JSON."""
```

Comment:
Please add a brief note that we are setting separators due to orjson doing so, too.

Reply:
Done